### PR TITLE
NAS-121620 / 23.10 / Add Terraria to `community` train

### PR DESCRIPTION
**Pull request by stavros-k**

### CODE REVIEW

File: library/ix-dev/community/terraria/questions.yaml [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+            min: 1
+            max: 255
+            required: true
+        - variable: password
+          label: Server Password
+          description: Sets the server password.
+          schema:
+            type: string
+            private: true
+            default: ''
```
Comment (Qubad786) [inline library/ix-dev/community/terraria/questions.yaml:122]: If this is required please have it as such and `empty=False`

File: library/ix-dev/community/terraria/upgrade_info.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"filename": "values.yaml", "keys": ["image", "vanillaImage"]}
```
[the remark this answers is not in the document]
Comment (Qubad786) [reply library/ix-dev/community/terraria/upgrade_info.json:1]: Let's talk on slack wrt this ✅ 
For now it is good

File: library/ix-dev/community/terraria/questions.yaml (hunk 1 above)
Comment (stavros-k) [reply library/ix-dev/community/terraria/questions.yaml:122]: It's optional!

**Comment by stavros-k** [comment]
I adjusted update_strategy to handle single image for now,
Opened a ticket to adjust it for more than 1
https://ixsystems.atlassian.net/browse/TNCHARTS-162

**Review by Qubad786 — APPROVED** [review]

**Pull request merged**